From 25a51134e6af976d9373b05821f12dde0011bbbe Mon Sep 17 00:00:00 2001
From: Terry Lv <r65388@freescale.com>
Date: Mon, 19 Mar 2012 16:50:54 +0800
Subject: [PATCH 198/256] ENGR00177165: Fix build error of missing videodev.h

Kernel now uses videodev2.h. Thus we need to change it.

Signed-off-by: Terry Lv <r65388@freescale.com>
---
 ipu/mxc_ipu_hl_lib_dummy.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ipu/mxc_ipu_hl_lib_dummy.c b/ipu/mxc_ipu_hl_lib_dummy.c
index 85f7933..d75f4b2 100644
--- a/ipu/mxc_ipu_hl_lib_dummy.c
+++ b/ipu/mxc_ipu_hl_lib_dummy.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2009-2011 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2009-2012 Freescale Semiconductor, Inc. All Rights Reserved.
  *
  */
 
@@ -37,7 +37,7 @@
 #include <sys/mman.h>
 #include <sys/types.h>
 #include <sys/stat.h>
-#include <linux/videodev.h>
+#include <linux/videodev2.h>
 #include "mxc_ipu_hl_lib.h"
 
 int mxc_ipu_lib_task_init(ipu_lib_input_param_t * input,
-- 
1.8.0

